src/content_scripts/chrome.js

```diff
@@ -0,0 +1,10 @@
+// communication type
+const UPDATE_CONTEXT_MENU = "updateContextMenu";
+
+document.addEventListener("selectionchange", () => {
+    const aselection = document.getSelection().toString();
+    browser.runtime.sendMessage({
+        type: UPDATE_CONTEXT_MENU,
+        selection: aselection
```

Comment:
I'm not a friend of `a` variables. `selection` is not a reserved name and it does not conflict with any variable, even when you use `selection` here, so just do that…

```suggestion
        selection: selection
```